conans/test/functional/command/search_test.py

```diff
@@ -247,6 +247,20 @@ class Test(ConanFile):
                         lib/1.0@foo/bar
                         lib/1.0@user/channel"""), client.out)
 
+    def search_disabled_remote_test(self):
+        self.client.run("remote disable search_able")
+        self.client.run("search * -r search_able", assert_error=True)
+        self.assertIn("ERROR: Remote 'search_able' is disabled", self.client.out)
+
+    def search_skip_disabled_remote_test(self):
+        os.rmdir(self.servers["local"].server_store.store)
+        self._copy_to_server(self.client.cache, self.servers["local"].server_store)
+        os.rmdir(self.servers["search_able"].server_store.store)
+        self._copy_to_server(self.client.cache, self.servers["search_able"].server_store)
+        self.client.run("remote disable local")
+        self.client.run("search Hello* -r all")
+        self.assertNotIn("Remote 'local':", self.client.out)
```

Comment:
Can we search just for `local`? Otherwise, if we change anything else in the string (quotes, add something before `:`,...) we might get a false positive and this test would be useless.